lisa/tools/ping.py

```diff
@@ -27,7 +27,7 @@ def can_install(self) -> bool:
         return True
 
     def install(self) -> bool:
-        if isinstance(self.node.os, Debian):
+        if isinstance(self.node.os, Ubuntu) or isinstance(self.node.os, Debian):
```

Comment:
If there is no difference on logic, it doesn't need to call out Ubuntu. The `isinstance` returns True, if it's Ubuntu, because the Ubuntu is a subclass of Debian.

Reply:
You're right. This issue ended up being a specific use of the ping command with node execute and not the Ping tool